src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProviderResponseValidation.java

```diff
@@ -21,6 +29,8 @@ public class EmbeddingProviderResponseValidation implements ClientResponseFilter
 
   static final MediaType MEDIATYPE_TEXT_JSON = new MediaType("text", "json");
 
+  Logger logger = LoggerFactory.getLogger(EmbeddingProvider.class);
```

Comment:
Usually should be the class itself, here `EmbeddingProviderResponseValidation.class`